api-testing/functional/tests/cypress/integration/01-institution/patch-institutions-institution.ts

```diff
@@ -0,0 +1,51 @@
+let identity: any;
+describe("Institution api", () => {
+  it("POST - Token", () => {
+    cy.fixture("authData").then((auth) => {
+      cy.request({
+        method: "POST",
+        url: Cypress.env("token_url"),
+        form: true,
+        body: {
+          grant_type: auth.grantType,
+          client_id: auth.clientId,
+          username: auth.userName,
+          password: auth.password,
+        },
+      }).then((response) => {
+        identity = response;
+        window.localStorage.setItem("identity", JSON.stringify(identity));
```

Comment:
If tests run in parallel we will need to keep the different tokens for different clients (e.g. students, institutions, AEST, workflow). It would be better to save them into different names, like the content of the `auth.clientId`.

Reply:
Updated.